### PR TITLE
Lint code examples

### PR DESCRIPTION
**Pull request by xtuc**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Arrow functions **are not** synonymous with normal functions. `arguments` and `t
 reference their *outer function* for example:
 
 ```javascript
-var user = {
+const user = {
```
Comment (xtuc) [inline docs/faq.md]: The eslint configuration prefers const. Given that most modern browser support `const`, I think that's fine.

Comment (hzoo) [inline docs/faq.md]: It's going to be run through Babel anyway though so definetely ok

**Comment by hzoo** [comment]
I guess we would need to run it on the markdown in babel/babel - everything getting more complicated haha

**Comment by xtuc** [comment]
Yes, this is the goal. I'm just POCing right now.

**Pull request merged**